plugins/wasi_nn/chattts.cpp

```diff
@@ -0,0 +1,361 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+#include "chattts.h"
+#include "wasinnenv.h"
+
+#ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_CHATTTS
+#include "simdjson.h"
+
+#if !defined(_WIN32) && !defined(_WIN64) && !defined(__WIN32__) &&             \
+    !defined(__TOS_WIN__) && !defined(__WINDOWS__)
+#include <dlfcn.h>
+#endif
+#include <chrono>
+#include <time.h>
+#endif
+
+namespace WasmEdge::Host::WASINN::ChatTTS {
+#ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_CHATTTS
+#if defined(_WIN32) || defined(_WIN64) || defined(__WIN32__) ||                \
+    defined(__TOS_WIN__) || defined(__WINDOWS__)
+HINSTANCE SharedLib = LoadLibrary(PYTHON_LIB_PATH);
+#else
+void *SharedLib = dlopen(PYTHON_LIB_PATH, RTLD_GLOBAL | RTLD_NOW);
+#endif
+Expect<WASINN::ErrNo> load(WASINN::WasiNNEnvironment &Env,
+                           Span<const Span<uint8_t>>, WASINN::Device,
+                           uint32_t &GraphId) noexcept {
+  // Add a new graph.
+  Env.NNGraph.emplace_back(Backend::ChatTTS);
+  auto &GraphRef = Env.NNGraph.back().get<Graph>();
+  // Initialize the plugin parameters.
+  GraphRef.EnableDebugLog = true;
```

Comment:
Looks like `EnableDebugLog` is always true. Do we have any way to change it? If not, we can remove this condition.
If you want to keep it, this variable should be one of the metadata keys.